web/client/actions/__tests__/locale-test.js

```diff
@@ -5,7 +5,6 @@
  * This source code is licensed under the BSD-style license found in the
```

Comment:
Every file you modify you have to change the date, in the copyright section, from 2015 to 2015-2016